### ISSUE TITLE
Cannot create plan

### ISSUE THREAD

**Issue by HasAndries** [recurly/recurly-client-net#226]
I cannot seem to create a plan, keep getting error **Unacceptable tag <trial_requires_billing_info>**. I suspect it's a bug and would appreciate assistance or bug fix.
Here's the relevant code pieces:
C#
```
var newPlan = false;
var planCode = plan.GetCode(billingInterval);
Plan recurlyPlan;
try
{
	recurlyPlan = Plans.Get(planCode);
}
catch (NotFoundException)
{
	recurlyPlan = new Plan(planCode, plan.Name);
	newPlan = true;
}
recurlyPlan.Description = plan.Description;
recurlyPlan.PlanIntervalLength = billingInterval == BillingIntervals.Annual ? 12 : 1;
recurlyPlan.PlanIntervalUnit = Plan.IntervalUnit.Months;
recurlyPlan.UnitAmountInCents["USD"] = (int)(plan.MonthlyPrice * 100);

if (newPlan) recurlyPlan.Create();
else recurlyPlan.Update();
```
HTTP Request:
```
POST https://???.recurly.com/v2/plans/ HTTP/1.1
Accept: application/xml
Content-Type: application/xml; charset=utf-8
User-Agent: Recurly C# Client v1.4.12.0
Authorization: Basic ???
X-Api-Version: 2.5
Host: ???.recurly.com
Content-Length: 481
Expect: 100-continue

<?xml version="1.0" encoding="utf-8"?>
<plan>
  <plan_code>LOBSTERINK-MONTHLY</plan_code>
  <name>Professional</name>
  <description>Advance your career using a learning platform designed for individual users</description>
  <plan_interval_unit>months</plan_interval_unit>
  <plan_interval_length>1</plan_interval_length>
  <unit_amount_in_cents>
    <USD>1499</USD>
  </unit_amount_in_cents>
  <trial_requires_billing_info>false</trial_requires_billing_info>
</plan>
```
HTTP Response:
```
HTTP/1.1 400 Bad Request
Date: Tue, 02 May 2017 11:13:31 GMT
Content-Type: application/xml; charset=utf-8
Transfer-Encoding: chunked
Connection: keep-alive
Set-Cookie: __cfduid=d6643c18c7969f2c13503f9aa5d9dc69f1493723611; expires=Wed, 02-May-18 11:13:31 GMT; path=/; domain=.recurly.com; HttpOnly
X-Api-Version: 2.5
Content-Language: en-US
X-RateLimit-Limit: 2000
X-RateLimit-Remaining: 1992
X-RateLimit-Reset: 1493723880
Cache-Control: no-cache
X-Request-Id: arp3eo35g6curc46amv0
Strict-Transport-Security: max-age=15552000; includeSubDomains; preload
Server: cloudflare-nginx
CF-RAY: 358a8d3bdaf172b9-AMS

<?xml version="1.0" encoding="UTF-8"?>
<error>
  <symbol>invalid_xml</symbol>
  <description>The provided XML was invalid.</description>
  <details>Unacceptable tag &lt;trial_requires_billing_info&gt;</details>
</error>
```

**Comment by bhelx** [comment]
@HasAndries which version of the client library are you using? I need to check, but I believe that field is only available in API version 2.6 but it looks like your version of the library is using 2.5.

**Comment by bhelx** [comment]
Sorry, seeing it now ` v1.4.12.0`

**Comment by bhelx** [comment]
Sorry @HasAndries, from what I can tell, I think we may have slipped this field in too early. I'll try to prepare a PR for tomorrow to fix this. In the mean time you should be safe to use `1.4.11` if you just want to get unblocked: https://github.com/recurly/recurly-client-net/releases

**Comment by HasAndries** [comment]
Thanks @bhelx for the quick response. 1.4.11 seems to work properly, will check back for the update.

**Comment by bhelx** [comment]
@HasAndries Thanks again for reporting and being so descriptive. If you get a chance please verify that #227 works for you. If this seems to fix the problem I can push out a version `1.4.13` today.

**Comment by bhelx** [comment]
@HasAndries this has gone out as `1.4.13` https://github.com/recurly/recurly-client-net/releases/tag/1.4.13

Please let us know if there are any other issues.
